tests/basic/test_basic_simd_vector.cpp

```diff
@@ -35,7 +35,9 @@
 // ==========================================================================
 
 #include <seqan/basic.h>
+#if SEQAN_SIMD_ENABLED
```

Comment:
I would rather move this macro into the file so that the file can be included in all situations...